source/sphinx_extensions/plugin_directory/templates/action.rst

```diff
@@ -32,11 +32,15 @@
      </ul>
      <div class="tab-content">
        <div id="cli" class="tab-pane fade in active">
-         <pre>
-           {{- cli_help -}}
-         </pre>
+         <h4>Invocation:</h4>
+         <pre>qiime {{plugin_cli_name}} {{action_cli_name}} [OPTIONS]</pre>
+         <h4>Docstring:</h4>
+         <pre>{{- cli_help -}}</pre>
        </div>
        <div id="api" class="tab-pane fade">
+       <h4>Import:</h4>
+       <pre>from qiime2.plugins.{{- plugin_api_name -}}.actions import {{action_api_name}} as {{action_api_name}}</pre>
```

Comment:
`{{- plugin_api_name -}}` --- the `-`s tacked onto the mustaches indicate that jinja should [strip leading and trailing whitespace](http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control) - is that necessary here? I wouldn't expect to see leading and trailing whitespace present in this value, but perhaps your development experience here indicated otherwise --- let me know!